tests/sentry/integrations/github_enterprise/test_issues.py

```diff
@@ -35,12 +36,13 @@ def setUp(self):
                 },
             )
             self.model.add_organization(self.organization, self.user)
-        self.integration = GitHubEnterpriseIntegration(self.model, self.organization.id)
+        install = self.model.get_installation(self.organization.id)
+        self.install = cast(GitHubEnterpriseIntegration, install)
 
     def _check_proxying(self) -> None:
         assert len(responses.calls) == 1
         request = responses.calls[0].request
-        assert request.headers[PROXY_OI_HEADER] == str(None)
+        assert request.headers[PROXY_OI_HEADER] == str(self.install.org_integration.id)
```

Comment:
this `assert` is now the same as the analogous one in `_check_proxying` in the github integration test, i don't think it should have been None in the first place D: